lib/params/XmlNode.cpp

```diff
@@ -412,8 +413,9 @@ const string &XmlNode::GetElementString(const string &tag) const {
 void XmlNode::GetElementStringVec(const string &tag, vector <string> &vec) const {
 	
 	string s = XmlNode::GetElementString(tag);
-
 	StrToWordVec(s, vec);
+    for (string &e : vec)
```

Comment:
Does this code do what we want? If I have a two element string vector:

a[0] = "string"
a[1] = "string with spaces"

that I encode with SetElementStringVec

will I get be able to get it back as a *two* element vector with GetElementStringVec()? It looks like I'll get four element vector back

Reply:
The previous behavior of `StrToWordVec` was to what you describe: you would
get a four element vector back. I changed `StrToWordVec` to only split at
non-escaped strings and when saving to params, spaces inside strings are
escaped then they are joined with non-escaped strings.

Reply:
OK. I overlooked the changes to StrToWordVec